Sources/RPListener.swift

```diff
@@ -172,6 +178,18 @@ public class RPListener: NSObject, XCTestObservation {
     }
   }
 
+  private func getServerName() -> String {
+    //Determine user name/machine name which inits test run. Initially, the data looks like /Users/epamcontractor/Library/Developer/...
```

Comment:
`ProcessInfo.processInfo.arguments[0].split(separator: "/")[1]` do the same thing

Reply:
This is genius!